classes/local/client/s3_client.php

```diff
@@ -133,6 +133,35 @@ public function upload_file(string $filepath, string $keyname): ?string {
         return $s3url;
     }
 
+    /**
+     * Returns a status message for the client.
+     *
+     * @return string
+     */
+    public function get_client_status_message(): string {
+        global $OUTPUT;
+
+        $output = '';
+        $connection = $this->test_connection();
+        if ($connection->success) {
+            $output .= $OUTPUT->notification(get_string('connectionsuccess', 'tool_s3logs'), 'notifysuccess');
```

Comment:
As I know, `notifysuccess` and `notifyproblem` are considered as legacy type of notification, see https://github.com/moodle/moodle/blob/master/lib/outputrenderers.php#L3016-L3029
I would swap to `success` and `error` respectively

Reply:
Good catch. Let's fix  them all in once as a separate PR.

Reply:
https://github.com/catalyst/moodle-tool_s3logs/issues/23